docs/guides/references/migration-guide.mdx

```diff
@@ -56,17 +56,29 @@ Related notes:
   configuration using `cypress.config.js` and `cypress.config.ts` vs the older
   `cypress.json`. For example:
 
-:::cypress-config-example
+<CypressConfigFileTabs>
```

Comment:
This is another example that in prod shows a deprecated 3rd tab with the .json version.